ec/acc/acc_prereq.go

```diff
@@ -67,3 +65,9 @@ func testAccPreCheck(t *testing.T) {
 		t.Fatal("Only one of API Key or Username / Password can be specified to execute acceptance tests")
 	}
 }
+
+func requiresAPIConn(t *testing.T) {
```

Comment:
Let's put a comment about what this function does so future contributors know when/how to use it :)